src/components/autocomplete/js/autocompleteController.js

```diff
@@ -491,7 +491,7 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
           ctrl.matches = [];
 
           setLoading(false);
-          reportMessages(false, ReportType.Count);
+          reportMessages(true, ReportType.Count);
```

Comment:
This stops the screen reader from interrupting the message about a chip being added if you have a long enough `md-chip-append-delay`. With the default delay, the chip added message is still interrupted on VoiceOver, but that's an issue for another time.